packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/Contracts.jsx

```diff
@@ -67,6 +67,19 @@ const SecondaryText = ({ contract }) => {
       </>
     )
   }
+  if (contract.metadata.disabledAt) {
+    return (
+      <>
+        <Icon icon={UnlinkIcon} size={8} />{' '}
```

Comment:
It's weird to put a space like that. No?

Reply:
Ca évite une surcharge du style rien que pour faire un margin de 3px, c'est une façon de faire qu'on a déjà ici et dans Banks. J'ai trouvé l'idée pas mal finalement